frameioclient/download.py

```diff
@@ -1,18 +1,27 @@
 import requests
-import math
 import os
 
 class FrameioDownloader(object):
-  def __init__(self, asset, download_folder):
+  def __init__(self, asset, download_folder, replace):
```

Comment:
Can you add a default for `replace` so that if the user doesn't specify (or just upgrades the package and runs it against old code), the downloader doesn't break?

Reply:
Definitely, I put it in client.py.
def download(self, asset, download_folder, replace=True). Is that okay or should we move it?

Reply:
> Definitely, I put it in client.py.
> def download(self, asset, download_folder, replace=True). Is that okay or should we move it?

Perfect!